### PR TITLE
Don't fetch perks for bots

### PR DESCRIPTION
**Pull request by 1zc**
Trying to fetch perks for bots is unnecessary, so this PR simply adds checks to prevent the plugin from doing so

### CODE REVIEW

File: scripting/GFL-UserManagement.sp [hunk 1]
```diff
@@ -140,23 +140,25 @@ public void OnClientDisconnect(int client) {
 }
 
 public void OnClientAuthorized(int client, const char[] sAuth2) {
-	// Get their Steam ID 64.
-	char steamID64[64];
-	GetClientAuthId(client, AuthId_SteamID64, steamID64, sizeof(steamID64), true);
+	if (!IsFakeClient(client)) {
```
Comment (NVedsted) [inline scripting/GFL-UserManagement.sp]: Imagine not early exiting `if (IsFakeClient(client)) return;`!!! Think of all the whitespace!!!!!!!!!!!!!!!!

Comment (Vauff) [reply scripting/GFL-UserManagement.sp]: +1

Comment (1zc) [reply scripting/GFL-UserManagement.sp]: I am a whitespace enjoyer and you will enjoy it too

**Review by Vauff — APPROVED** [review]

**Review by NVedsted — APPROVED** [review]
Truly a sight to behold.

**Pull request merged**